src/components/common/AppBars/Navbar.js

```diff
@@ -0,0 +1,264 @@
+import React, { Component } from "react";
+import { connect } from "react-redux";
+import PropTypes from "prop-types";
+import moment from "moment";
+import checked from "../../../assets/icons/check.svg";
+import circle from "../../../assets/icons/fire.svg";
+import close from "../../../assets/icons/close.svg";
+
+import {
+  fetchNotifications,
+  readNotification,
+  deleteNotification
+} from "../../../redux/actions/notificationActions";
+
+/* eslint global-require: "off" */
+
+export class Navbar extends Component {
+  constructor(props) {
```

Comment:
No need of this constructor, you can declare state directly.

Reply:
Okay, Thanks for the feedback  I'm gonna remove it in the next push